src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> reports = new HashMap<>();
```

Comment:
Consider making the `reports` map non-static and providing access to it via instance methods. Static fields can lead to issues with testability and reusability of the class. Additionally, check if 'reports' is the best name for this map, as it might not clearly represent its purpose.